crates/sui-graphql-rpc/src/types/transaction_block/tx_lookups.rs

```diff
@@ -0,0 +1,461 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    data::{pg::bytea_literal, Conn, DbConnection},
+    filter, inner_join, query,
+    raw_query::RawQuery,
+    types::{
+        cursor::Page,
+        digest::Digest,
+        sui_address::SuiAddress,
+        transaction_block::TransactionBlockKindInput,
+        type_filter::{FqNameFilter, ModuleFilter},
+        uint53::UInt53,
+    },
+};
+use diesel::{
+    query_dsl::positional_order_dsl::PositionalOrderDsl, CombineDsl, ExpressionMethods,
+    NullableExpressionMethods, QueryDsl,
+};
+use std::fmt::Write;
+use sui_indexer::schema::checkpoints;
+use sui_types::base_types::SuiAddress as NativeSuiAddress;
+
+use super::{Cursor, TransactionBlockFilter};
+
+#[derive(Clone, Debug, Copy)]
+pub(crate) struct TxBounds {
```

Comment:
It would be useful to comment whether these bounds are inclusive or exclusive on each side.